src/Exception/BadRequestHttpException.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Exception;
+
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException as BaseBadRequestHttpException;
+
+/**
+ * This exception is thrown when there is an issue with the request.
+ *
+ * @author Christian Schiffler <https://github.com/discordier>
+ */
+class BadRequestHttpException extends BaseBadRequestHttpException
```

Comment:
But that would mean our pretty error screen is not rendered for "regular" exceptions. I think that's wrong, we should render our error 500 pretty screen for everything that is not catched.